### PR TITLE
🏗 Collect z-index from JS files

### PR DESCRIPTION
**Pull request by alanorozco**
`z-index` can be set on JS files:

```js
setStyle(element, 'z-index', 1);
setStyles(element, {'z-index': 1});

<div style={{zIndex: 30}} />

export const JSS = {
  className: {
    zIndex: 'auto',
  },
};

```

This allows `gulp get-zindex` to collect these instances.

List is also now formatted with link.

### CODE REVIEW

**Review by rsimha — APPROVED** [review]

File: build-system/tasks/get-zindex/index.js [hunk 1]
```diff
@@ -115,11 +122,52 @@ async function getZindexSelectors(glob, cwd = '.') {
   return filesData;
 }
 
+/**
+ * @param {string|Array<string>} glob
+ * @param {string=} cwd
+ * @return {Object}
+ */
+async function getZindexChainsInJs(glob, cwd = '.') {
```
Comment (erwinmombay) [inline build-system/tasks/get-zindex/index.js:130]: qq, why i dont see any await in the body of the function, why did this have to be async?

Comment (erwinmombay) [reply build-system/tasks/get-zindex/index.js:130]: nevermind. is it the `tempy.write` task i assume?

Comment (erwinmombay) [inline build-system/tasks/get-zindex/index.js:130]: would this still work for references or does the value have to be on the call site? (apologies, still trying to read through the code)

Comment (alanorozco) [reply build-system/tasks/get-zindex/index.js]: The `tempy.write` task returns a promise, but this function doesn't need to be async.

Comment (alanorozco) [reply build-system/tasks/get-zindex/index.js]: It will not follow references.

Comment (erwinmombay) [reply build-system/tasks/get-zindex/index.js]: do we have any lint rules to enforce it?

**Review by erwinmombay — APPROVED** [review]

File: build-system/tasks/get-zindex/index.js (hunk 1 above)
Comment (alanorozco) [reply build-system/tasks/get-zindex/index.js]: No. I haven't encountered it so far.

**Pull request merged**